### PR TITLE
[ZH-TW] Add `OWC/2023`

### PR DESCRIPTION
**Pull request by rucker122**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Review by deyomoe — APPROVED** [review]

File: wiki/Tournaments/OWC/2023/zh-tw.md [hunk 1]
```diff
@@ -0,0 +1,426 @@
+---
+tags:
+  - OWC
+  - OWC2023
+---
+
+# osu! World Cup 2023
+
+![OWC 2023 banner](img/owc2023-banner.jpg)
```
Comment (cl8n) [inline wiki/Tournaments/OWC/2023/zh-tw.md:9]: the alt text (`OWC 2023 banner`) should be translated

File: wiki/Tournaments/OWC/2023/zh-tw.md [hunk 2]
```diff
@@ -0,0 +1,426 @@
...
+
+## 獎項
+
+主辦方提供美金 $5,000 的獎金以及限定獎品。
+
+| 名次 | 獎賞 |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "冠軍") | 48% 的獎金、限量發行獎品、個人頁面獎章、一年的 **osu! Champion** 個人稱號 |
+| ![Silver crown](/wiki/shared/crown-silver.png "亞軍") | 32% 的獎金、限量發行獎品、個人頁面獎章 |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "季軍") | 20% 的獎金、限量發行獎品、個人頁面獎章 |
```
Comment (cl8n) [inline wiki/Tournaments/OWC/2023/zh-tw.md:35]: likewise, gold/silver/bronze crown should be translated

File: wiki/Tournaments/OWC/2023/zh-tw.md [hunk 3]
```diff
@@ -0,0 +1,426 @@
...
+  2. [A-One feat. Shihori - Bamboo Dance (nanoya) \[Lunatic\]](https://osu.ppy.sh/beatmapsets/2075280#osu/4344475)
+  3. [Sable Hills - Not Falling (squirrelpascals) \[Vid\]](https://osu.ppy.sh/beatmapsets/2075253#osu/4344422)
+
+## 比賽結果
+
+### 32 強
+
+2023 年 10 月 28 日, 星期六：
+
+| A 隊 |  |  | B 隊 | Match link | VOD link |
```
Comment (cl8n) [inline wiki/Tournaments/OWC/2023/zh-tw.md:199]: i copied this table over from EN, but I couldn't find existing translations for "Match link" or "VOD link"

**Review by cl8n — CHANGES_REQUESTED** [review]
everything else looks good :ok_hand:

File: wiki/Tournaments/OWC/2023/zh-tw.md (hunk 3 above)
Comment (rucker122) [reply wiki/Tournaments/OWC/2023/zh-tw.md]: "Mp link" and "VOD" are common in the community, but i put formal words here.

File: wiki/Tournaments/OWC/2023/zh-tw.md (hunk 2 above)
Comment (cl8n) [reply wiki/Tournaments/OWC/2023/zh-tw.md:35]: still need this one :eyes:

Comment (rucker122) [reply wiki/Tournaments/OWC/2023/zh-tw.md]: My bad, fixed. :)

**Review by cl8n — APPROVED** [review]

**Pull request merged**